README.md

```diff
@@ -1,54 +1,74 @@
-# Homebrew Tap Repository for Hazelcast Command Line
+# Homebrew Tap Repository for Hazelcast Platform
 
-`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) and [Management Center](https://hazelcast.org/imdg/download/#hazelcast-imdg-management-center) on local environment.
+`hz` is a tool which allows users to install & run [Hazelcast IMDG](https://hazelcast.org/imdg/) on a local environment.
```

Comment:
IMDG?

Reply:
hmm, I don't follow, should I use `Hazelcast Platform` instead?

Reply:
changed: https://github.com/hazelcast/homebrew-hz/pull/27/commits/138cc6e9f9dc3a74f033fb613c85fa1e881c1449